libraries/apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/ApolloResponse.kt

```diff
@@ -96,6 +96,8 @@ private constructor(
 
   fun hasErrors(): Boolean = !errors.isNullOrEmpty()
 
+  fun isSuccessful(): Boolean = exception == null && !hasErrors()
```

Comment:
I'm still not sure about this one. Is a "partial response" successful or not? I don't want to hurt their feelings 😅 . A partial success is still some sort of success....